layouts/base.njk

```diff
@@ -45,6 +45,8 @@
   <link rel="stylesheet" href="{{ stylesheet | canonicalUrl }}">
   {% endfor %}
 
+  {% if options.feedUrl %}<link rel="alternate" type="application/atom+xml" href="{{ options.feedUrl }}" >{% endif %}
```

Comment:
Add canonical URL so that if a relative path is provided it gets properly resolved.

```suggestion
  {% if options.feedUrl %}<link rel="alternate" type="application/atom+xml" href="{{ options.feedUrl | canonicalUrl }}" >{% endif %}
```